ONSdigital/response-operations-social-ui
# Use the latest SDC pattern library for Response Operations Social UI

# Motivation and Context

Update Response Operations Social UI service to use components and patterns from the [SDC pattern library.](https://sdc-global-design-patterns.netlify.com/index.html)

# What has changed
- CSS removed that is also present in the pattern library
- Minimised about of required local css
- Removed all unused JS
- Removed unused templates
- A number of templates refactored to use pattern library markup
- Improved code generation UI to display new code in panel at top of reloaded page, rather than out of view at the bottom
- Tidied up indentation and whitespace in templates
- Updated some page titles to be more task centric and follow a consistent pattern across the service.

# How to test?
- Ensure the response-ops-social repo will run
- Ensure the response-ops-social repo units tests pass
- Run through all SEL UI user journeys to make sure that no UI regressions have occurred, and no journeys now fail.

# Links
https://trello.com/c/yCnRVGM8/326-update-sel-ui-to-use-the-pattern-library-cdn
